src/components/atoms/Dialog/Dialog.tsx

```diff
@@ -53,20 +53,27 @@ function DialogContent({
   return (
     <DialogPortal data-slot="dialog-portal">
       <DialogOverlay />
-      <div className="fixed inset-0 z-50 flex items-center justify-center px-4 py-6 sm:px-0 sm:py-0">
+      <div className="fixed inset-0 z-50 flex items-center justify-center">
         <DialogPrimitive.Content
           data-slot="dialog-content"
           data-testid="dialog-content"
           className={Libs.cn(
-            'relative z-50 grid w-full max-h-[80vh] overflow-y-auto rounded-lg border bg-background p-6 shadow-lg duration-200 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[state=open]:animate-in data-[state=open]:fade-in-0 data-[state=open]:zoom-in-95 sm:max-w-lg',
+            'relative z-50 grid' +
+              ' overflow-y-auto rounded-lg border bg-background shadow-lg duration-200' +
+              ' data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95' +
+              ' data-[state=open]:animate-in data-[state=open]:fade-in-0 data-[state=open]:zoom-in-95 p-6 sm:p-8 gap-6 m-4',
```

Comment:
This Libs.cn already concatenates all classes, so we need to use `,` instead of `+`
```suggestion
            'relative z-50 grid',
              ' overflow-y-auto rounded-lg border bg-background shadow-lg duration-200',
              ' data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95',
              ' data-[state=open]:animate-in data-[state=open]:fade-in-0 data-[state=open]:zoom-in-95 p-6 sm:p-8 gap-6 m-4',
```

Reply:
The use of `overflow-y-auto` here is wrong. we should always make the content respect the grid without any overflow.